ui/pages/swaps/countdown-timer/timer-icon.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function TimerIcon() {
+  return (
+    <svg
+      width="15"
+      height="15"
+      viewBox="0 0 15 15"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M7.49835 0.510498C3.75226 0.510498 0.717102 3.54565 0.717102 7.29175C0.717102 11.0378 3.75226 14.073 7.49835 14.073C11.2444 14.073 14.2796 11.0378 14.2796 7.29175C14.2796 3.54565 11.2444 0.510498 7.49835 0.510498ZM7.49835 12.7605C4.4632 12.7605 2.0296 10.3269 2.0296 7.29175C2.0296 4.28394 4.4632 1.823 7.49835 1.823C10.5062 1.823 12.9671 4.28394 12.9671 7.29175C12.9671 10.3269 10.5062 12.7605 7.49835 12.7605ZM9.16632 9.91675C9.33038 10.0261 9.52179 9.99878 9.63116 9.83472L10.1507 9.15112C10.2601 8.98706 10.2327 8.79565 10.0687 8.68628L8.26398 7.34644V3.46362C8.26398 3.29956 8.09991 3.1355 7.93585 3.1355H7.06085C6.86945 3.1355 6.73273 3.29956 6.73273 3.46362V7.97534C6.73273 8.05737 6.76007 8.16675 6.8421 8.22144L9.16632 9.91675Z"
+        fill="#037DD6"
+      />
+    </svg>
+  );
+}
```

Comment:
Our code base is inconsistent in use of svg icons. It will be useful to get this component generation automated.

Reply:
It would be great to have a unified way across the extension to deal with .svg files and have the ability to make them easily reusable across different features. Maybe we already have some best practices for dealing with .svg files?